### PR TITLE
ast(init): fix init command

### PR DESCRIPTION
**Pull request by ev1stensberg**
**What kind of change does this PR introduce?**

**Did you add tests for your changes?**

**If relevant, did you update the documentation?**

**Summary**

**Does this PR introduce a breaking change?**

**Other information**

### CODE REVIEW

File: lib/init/transformations/index.js [hunk 1]
```diff
@@ -111,15 +113,16 @@ module.exports = function runTransform(webpackProperties, action) {
 		})
 			.then(_ => {
 				let configurationName;
-				if (!config.configName) {
+				if (!config.configName && action !== "init") {
```
Comment (ematipico) [inline lib/init/transformations/index.js]: You could save the expression `action !== "init"` in variable
```js
const isActionNotInit = action !== "init";
```

File: package.json [hunk 2]
```diff
@@ -31,13 +31,11 @@
     "travis:integration": "yarn prepare && nyc jest && yarn reportCoverage",
     "travis:lint": "yarn lint && yarn bundlesize",
     "bundlesize": "bundlesize",
-    "commitmsg": "commitlint -e $GIT_PARAMS",
```
Comment (ematipico) [inline package.json]: Why this has been removed?

**Comment by ematipico** [comment]
@ev1stensberg is there an issue to understand what we are trying to fix with this PR?

File: package.json (hunk 2 above)
Comment (ev1stensberg) [reply package.json]: errors even though commit is correct

File: lib/init/transformations/index.js (hunk 1 above)
Comment (ev1stensberg) [reply lib/init/transformations/index.js]: its used one time, so why bother?

**Comment by ev1stensberg** [comment]
Yeah, #274

File: lib/init/transformations/index.js (hunk 1 above)
Comment (ematipico) [reply lib/init/transformations/index.js]: No it is not, I see twice in the code changes.

**Pull request merged**